config.js

```diff
@@ -3,188 +3,23 @@ const templateData =
-                link: ""
-              },
-              {
-                title: "change",
-                link: ""
-              },
-              {
-                title: "check",
-                link: ""
-              },
-              {
-                title: "click",
-                link: ""
-              },
-              {
-                title: "submit-click",
-                link: ""
-              },
-              {
-                title: "update",
-                link: ""
-              },
-              {
-                title: "timer",
-                link: ""
-              },
-              {
-                title: "redirect",
-                link: ""
-              },
-              {
-                title: "copy",
-                link: ""
-              },
-              {
-                title: "pause",
-                link: ""
-              }
-            ]
-          },
-          {
-            title: "Functions",
-            link: "",
-            sub_items: [
-              {
-                title: "clear cookies",
-                link: ""
-              }
-            ]
-          },
-          {
-            title: "Recording",
-            link: ""
-          },
-          {
-            title: "Play",
-            link: ""
-          },
-          {
-            title: "Clipboard",
-            link: ""
-          },
-          {
-            title: "Options",
-            link: "",
-            sub_items: [
-              {
-                title: "CBA",
-                link: ""
-              },
-              {
-                title: "Functions Option",
-                link: ""
-              },
-              {
-                title: "Import-Export",
-                link: ""
-              }
-            ]
-          }
-        ]
-      },
-      {
-        title: "Tutorial",
-        link: "#",
-        sub_items: [
-          {
-            title: "Registration Demo",
-            link: ""
-          },
-          {
-            title: "Facebook Demo",
-            link: ""
-          },
-          {
-            title: "Recording",
-            link: ""
-          },
-          {
-            title: "Video Presentation for CBA",
-            link: ""
-          }
-        ]
-      },
-      {
-        title: "Ready Projects",
-        link: "#",
-        sub_items: [
-          {
-            title: "FB like/unlike",
-            link: ""
-          }
-        ]
-      },
-      {
-        title: "What's New",
-        link: "#",
-        sub_items: [
-          {
-            title: "Version 8.0",
-            link: ""
-          },
-          {
-            title: "Version 6.0",
-            link: ""
-          },
-          {
-            title: "Version 5.0",
-            link: ""
-          }
-        ]
+    menuItems: ["", "documentation", "tutorial", "ready-projects",
+      "what's-new"],
+    pagesList: function (absolutePath) {
```

Comment:
Suggestion: Currently you are hardcoding page's `originalPathname` in EJS files, what you can do instead is pass `pathname` and get `originalPathname` using `site.queryPages `:
```js
function (pathname) {
  const query = templateData.site.queryPages;
  const {originalPathname} = query((page) => page.pathname === pathname)[0];
```